internal/provider/provider.go

```diff
@@ -66,7 +66,7 @@ func nameFromResourceData(nameMap map[string]interface{}) (*pkix.Name, error) {
 		result.SerialNumber = value.(string)
 	}
 
-	return result, nil
+	return result
```

Comment:
When `if` is false above, does the initialized/empty `result` signify an error somehow?

Reply:
I think the signature was written BEFORE the implementation was put down.

In a certificate, all those fields are optional, so it return an empty `result` and it should be ok.